cmd/crc/cmd/client.go

```diff
@@ -0,0 +1,18 @@
+package cmd
+
+import "github.com/code-ready/crc/pkg/crc/machine"
+
+type client interface {
+	Status(statusConfig machine.ClusterStatusConfig) (machine.ClusterStatusResult, error)
+	Exists(name string) (bool, error)
+}
+
+type libmachineClient struct{}
+
+func (*libmachineClient) Status(statusConfig machine.ClusterStatusConfig) (machine.ClusterStatusResult, error) {
+	return machine.Status(statusConfig)
+}
```

Comment:
@guillaumerose this new `client` for the cmd is only for unit test purpose so that it can be mocked and also extended for other commands ( like start,stop ..etc.) ?

Reply:
Yes !